test/api/OperatorApi/orderStatus.test.ts

```diff
@@ -0,0 +1,208 @@
+    })
+    test('Partially filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '11' }
+
+      expect(getOrderStatus(order)).toEqual('partially filled')
+    })
+  })
+})
+
+describe('Expired status', () => {
+  beforeEach(mockTimes)
+
+  describe('Buy order', () => {
+    test('Not expired, open', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'buy',
+        buyAmount: '10000',
+        executedBuyAmount: '0',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).not.toEqual('expired')
+    })
+    test('Expired', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'buy',
+        buyAmount: '10000',
+        executedBuyAmount: '0',
+        validTo: Math.floor(DATE.getTime() / 1000) - 1,
+      }
+      expect(getOrderStatus(order)).toEqual('expired')
+    })
+  })
+  describe('Sell order', () => {
+    test('Not expired, open', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'sell',
+        sellAmount: '10000',
+        executedSellAmount: '0',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).not.toEqual('expired')
+    })
+    test('Expired', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'sell',
+        sellAmount: '10000',
+        executedSellAmount: '0',
+        validTo: Math.floor(DATE.getTime() / 1000) - 1,
+      }
+      expect(getOrderStatus(order)).toEqual('expired')
+    })
+  })
+})
+
+describe('Open status', () => {
+  beforeEach(mockTimes)
+
+  describe('Buy order', () => {
+    test('Open, no fills', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'buy',
+        buyAmount: '10000',
+        executedBuyAmount: '0',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).toEqual('open')
+    })
+    test('Open, with partial fills', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'buy',
+        buyAmount: '10000',
+        executedBuyAmount: '10',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).toEqual('open')
+    })
+  })
+  describe('Sell order', () => {
+    test('Open, no fills', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'sell',
+        sellAmount: '10000',
+        executedSellAmount: '0',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).toEqual('open')
+    })
+    test('Open, with partial fills', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'sell',
+        sellAmount: '10000',
+        executedSellAmount: '10',
+        validTo: Math.floor(Date.now() / 1000),
+      }
+      expect(getOrderStatus(order)).toEqual('open')
+    })
+  })
+})
```

Comment:
I think there's nice to add cases where you try that, "sellAmount" don't affect the state for a buy order (and the other way around for sell orders)

Reply:
Done